python_modules/dagster/dagster/_grpc/impl.py

```diff
@@ -286,30 +282,16 @@ def get_external_sensor_execution(
     sensor_def = repo_def.get_sensor_def(sensor_name)
 
     with ExitStack() as stack:
-
-        if isinstance(sensor_def, MultiAssetSensorDefinition):
-            sensor_context = stack.enter_context(
-                MultiAssetSensorEvaluationContext(
-                    instance_ref,
-                    last_completion_time=last_completion_timestamp,
-                    last_run_key=last_run_key,
-                    cursor=cursor,
-                    repository_name=repo_def.name,
-                    repository_def=repo_def,
-                    asset_selection=sensor_def.asset_selection,
-                    asset_keys=sensor_def.asset_keys,
-                )
-            )
-        else:
-            sensor_context = stack.enter_context(
-                SensorEvaluationContext(
-                    instance_ref,
-                    last_completion_time=last_completion_timestamp,
-                    last_run_key=last_run_key,
-                    cursor=cursor,
-                    repository_name=repo_def.name,
-                )
+        sensor_context = stack.enter_context(
```

Comment:
still wrapping me head around some of the impl in this PR, can you explain why we don't need to create the MultiAssetSensorEvalContext anymore?

Reply:
It now gets created inside the evaluation function of `MultiAssetSensorDefinition`: https://github.com/dagster-io/dagster/pull/10258/files#diff-91398a61b68ddb85fac7e71bfb7d65188508c60968ea83a2f56c83d6e7791c6cR1120